.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: go
+
+go:
+  - 1.4
+  - tip
+
+env:
+  - GOTAGS=restapi
+  #- GOTAGS=powershell
+
+install:
+  # trick to make go find packer-azure packages under MSOpenTech
+  - ln -s paulmey ../../MSOpenTech

```

Comment:
also that's not how symlinks work. I believe the first arg (i.e. paulmey) should be a full path. 

try this:

```sh
$ mkdir a && cd a
$ mkdir b
$ ln -s b ../blink
$ cd ../
$ readlink blink
$ cd blink # wont work
no such file or directory: blink
```

try making it $PWD/paulmey instead